cmd/web/assets/css/input.css

```
@@ -0,0 +1,4 @@
+@import url("theme.css");
+
+@layer base {
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)